src/composables/Project.js

```diff
@@ -38,18 +47,27 @@ export function getProjectById(projectId) {
   const projects = getProjects();
   return projects[projectId];
 }
-
 /**
  * Save the project state.
  *
- * @param {Project} project
+ * @param {Project} project - Project to save
  */
 export function saveProject(project) {
   const projects = getProjects();
   projects[project.id] = project;
   localStorage.setItem(PROJECT_STORAGE_KEY, JSON.stringify(projects));
 }
 
+/**
+ * Save project and initialize git in local storage.
+ * @param {Project} project - Project to save
+ * @return {Promise<void>} Promise with nothing on success otherwise an error.
+ */
+export function createProject(project) {
```

Comment:
Explicitely declare the function as async, even if it only passes a promise from the underlying library.
It probably also should return directly the newly created project.

Reply:
No, I prefere to have all function that return a Promise like that you can make an async/await in your code when you call the function in your code :)

Reply:
Documentation indicated that's this function return promise.
Wo don't need to return project but I will rename this function te be more understandable :)